bootloader/src/main.rs

```diff
@@ -359,8 +372,10 @@ async fn jump_to_application(mut uart: Uart) -> ! {
 
             // We need to disable all used peripherals
             drop(uart);
-
-            unsafe { cortex_m::asm::bootload(application_address as *const u32) }
+            unsafe {
+                scb.vtor.write(application_address);
```

Comment:
I'm not entirely sure this is the correct address

Reply:
Yes that should be right